app/javascript/packages/document-capture/components/in-person-troubleshooting-options.tsx

```diff
@@ -48,11 +46,6 @@ function InPersonTroubleshootingOptions({
             text: t('idv.troubleshooting.options.learn_more_verify_by_phone_in_person'),
             isExternal: true,
           },
-          spName && {
-            url: getFailureToProofURL(location),
```

Comment:
It looks like we don't have any more usage of `getFailureToProofURL` and it could be removed from `ServiceProviderContextProvider`.